src/layout/section/landingGridSection/LandingGridSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { HeroSection, HeroSectionProps } from '../heroSection'
+import { LandingGridSectionElements } from './LandingGridSection.style'
+
+export type LandingGridSectionProps = HeroSectionProps &
+  Readonly<{
+    children: React.ReactNode
+  }>
+
+export const LandingGridSection = ({
```

Comment:
I would drop the Landing* from the name too. The component name should be descriptive of what it's doing not where it's used. We could have a quick async brainstorm for a good name in slack and other team members.

Reply:
This is really a grid only for these pages. I've even thought about naming it Page instead. But sure.. open for naming conventions

Reply:
I've followed @guillaumewuip  [naming suggestion ](https://github.com/blablacar/ui-library/pull/1065/commits/69e4198ccf19034c72563e2762b99240d8fa1c93)

Let me know if it that ok for you @grosbouddha